packages/yarnpkg-builder/sources/commands/build/plugin.ts

```diff
@@ -128,6 +128,15 @@ export default class BuildPluginCommand extends Command {
           minify: !this.noMinify,
           sourcemap: this.sourceMap ? `inline` : false,
           target: `node${semver.minVersion(pkg.engines.node)!.version}`,
+          supported: {
+            /*
+            Yarn plugin-runtime did not support builtin modules prefixed with "node:".
+            See https://github.com/yarnpkg/berry/pull/5997
+            As a solution, and for backwards compatibility, esbuild should strip these prefixes.
+            */
+            'node-colon-prefix-import': false,
+            'node-colon-prefix-require': false,
```

Comment:
Are you sure Esbuild will strip the `node:` prefix? I don't see that happening on the REPL: https://esbuild.github.io/try/#dAAwLjIxLjUALS10YXJnZXQ9bm9kZTIwCi0tc3VwcG9ydGVkOm5vZGUtY29sb24tcHJlZml4LWltcG9ydD1mYWxzZQotLXN1cHBvcnRlZDpub2RlLWNvbG9uLXByZWZpeC1yZXF1aXJlPWZhbHNlCi0tc291cmNlbWFwAGNvbnN0IHggPSByZXF1aXJlKCdub2RlOmZzJyk

Reply:
yepp. tretty sure. tested it with a local build.
i am not certain how a regression test for this could look like, since i am not familiar with the setup here.

Reply:
here is the rest: <https://esbuild.github.io/try/#YgAwLjIxLjUAewogIGJ1bmRsZTogdHJ1ZSwKICBmb3JtYXQ6ICdpaWZlJywKICBwbGF0Zm9ybTogJ25vZGUnLAogIHRhcmdldDogWydub2RlMjAnXSwKICBzdXBwb3J0ZWQ6IHsKICAgICdub2RlLWNvbG9uLXByZWZpeC1pbXBvcnQnOiBmYWxzZSwKICAgICdub2RlLWNvbG9uLXByZWZpeC1yZXF1aXJlJzogZmFsc2UsCiAgfSwKfQBlAGVudHJ5LmpzAGltcG9ydCBwYXRoIGZyb20gJ25vZGU6cGF0aCcKCnBhdGguam9pbignYScsICdiJyk>

Reply:
@arcanis is the re anything you need from me?

Reply:
@arcanis transform doesn't but build does.